util/logs/log-k8s-requests.go

```diff
@@ -17,7 +17,7 @@ func (m k8sLogRoundTripper) RoundTrip(r *http.Request) (*http.Response, error) {
 	x, err := m.roundTripper.RoundTrip(r)
 	if x != nil {
 		verb, kind := k8s.ParseRequest(r)
-		log.Infof("%s %s %d", verb, kind, x.StatusCode)
+		log.Debugf("Made kubernetes request %s %s, response %d", verb, kind, x.StatusCode)
```

Comment:
Also in general, I think we should avoid logs without some sort of searchable keyword -- i.e., `"%s %s %d"` is not trivial to find when seeing the output log.